.github/workflows/test_security.yml

```diff
@@ -57,6 +55,7 @@ jobs:
           then
             echo "FROM opensearchstaging/opensearch:$docker_version" >> Dockerfile
             echo "RUN if [ -d /usr/share/opensearch/plugins/opensearch-anomaly-detection ]; then /usr/share/opensearch/bin/opensearch-plugin remove opensearch-anomaly-detection; fi" >> Dockerfile
+            echo "RUN if [ -d /usr/share/opensearch/plugins/opensearch-time-series-analytics ]; then /usr/share/opensearch/bin/opensearch-plugin remove opensearch-time-series-analytics; fi" >> Dockerfile
```

Comment:
This implies a plugin name change that I imagine could have lots of other impacts, like infra builds / scripts or frontend plugin builds / scripts that rely on this. We should consider this a breaking change I don't believe we want in 2.x, lmk what you think.

I also see lots of changes related to cluster settings and REST APIs. Could you confirm those are all internal facing and nothing changes from an external/naming perspective? If the latter, same concern as above.

Reply:
> We should consider this a breaking change I don't believe we want in 2.x, lmk what you think.
I have already synced with infra team and will work with them after 2.13 release.

> Could you confirm those are all internal facing and nothing changes from an external/naming perspective? 
Those are internal facing.

Reply:
Sounds good. Let's call out the only change is related to the plugin name, rest of the functionality remains the same. Sounds like infra will be handled, can you handle tracking of changes on frontend plugin CIs?